nautobot/dcim/filters/__init__.py

```diff
@@ -1639,10 +1645,14 @@ class PowerPanelFilterSet(LocatableModelFilterSetMixin, NautobotFilterSet):
         field_name="power_feeds",
         label="Has power feeds",
     )
+    has_feeders = RelatedMembershipBooleanFilter(
+        field_name="feeders",
+        label="Has feeders",
+    )
 
     class Meta:
         model = PowerPanel
-        fields = ["id", "name", "tags"]
+        fields = ["id", "name", "panel_type", "breaker_position_count", "tags"]
```

Comment:
I don't *think* this addition is necessary, as IIRC these are just the fields that should be auto-derived, and you have both of the above explicitly declared above. But see above comment about `breaker_position_count` specifically.